File: tickets-agente-backend/src/main/java/yavirac/tickets/feature/motivo/Motivo.java
package yavirac.tickets.feature.motivo;


import java.sql.Timestamp;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.Table;

import lombok.Data;

@Data
@Table("motivos")
public class Motivo {
    
    
    @Id
    @Column("motivo_id")
    private long motivoId;

    private String descripcion;
    private Timestamp created;
    private Timestamp update;
    private boolean enabled;
    private boolean archived;

}

File: tickets-agente-frontend/src/app/feature/motivo/motivo.ts
import { Sugerencia } from "../sugerencia/sugerencia";

export interface Motivo {
    motivoId: number,
    descripcion: string,
    created: Date,
    enabled: boolean,
    archived: boolean,
    moduloId: number,
    sugerencias: Sugerencia[],
    moduloNombre: string
}
